bundle.Dockerfile

```diff
@@ -6,6 +6,15 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=gitops-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=stable
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
```

Comment:
Do we need these changes ?

Reply:
These were added automatically after running the script to build operator-index, wasn't sure if they're important so left them in
I can remove them though